src/services/cssNavigation.ts

```diff
@@ -422,7 +422,20 @@ export class CSSNavigation {
 			return this.mapReference(await this.resolveModuleReference(target, documentUri, documentContext), isRawLink);
 		}
 
-		const ref = await this.mapReference(documentContext.resolveReference(target, documentUri), isRawLink);
+		// Treat bare module names (“bootstrap/...”) like sass-loader does
+		const isBareImport = !target.startsWith('.')             // not ./ or ../
+		                 && !target.startsWith('/')              // not workspace-absolute
+		                 && target.indexOf(':') === -1;          // not a scheme (file://)
```

Comment:
Maybe a use a regex to make that check [
startsWithSchemeRegex = /^\w[\w\d+.-]:/